app/controllers/orders_controller.rb

```diff
@@ -0,0 +1,8 @@
+class OrdersController < ApplicationController
+	def index; end
+
+	def show
+		@order = Order.find(params[:id])
```

Comment:
Indentation in this file looks weird. Most Ruby developers use 2 spaces indentation level